examples/image/compute_show.py

```diff
@@ -24,7 +23,7 @@
 ###############################################################################
 # We can briefly visualize the data to understand the type of images we have.
 for library_id in adata.uns["spatial"].keys():
```

Comment:
use sq.pl.spatial_scatter instead with proper library_key instead of the for loop.

Reply:
Fixed.